src/wallet/fees.cpp

```diff
@@ -53,6 +53,9 @@ CAmount GetMinimumFee(unsigned int nTxBytes, const CCoinControl& coin_control, c
             // if we don't have enough data for estimateSmartFee, then use fallbackFee
             fee_needed = CWallet::fallbackFee.GetFee(nTxBytes);
             if (feeCalc) feeCalc->reason = FeeReason::FALLBACK;
+
+            // directly return if fallback fee is disabled (feerate 0 == disabled)
+            if (CWallet::fallbackFee.GetFee(1000) == 0) return fee_needed;
```

Comment:
shorter, faster and clearer:

```cpp
if (fee_needed == 0)

Reply:
fee_needed = 0 seems the wrong check, it could be 0 in case you would run through with `nTxBytes=0`, very unlikely, but testing the fallbackfee instead the current `fee_needed` seems correct.

Reply:
Right, should only matter for unit tests, but makes sense to keep it that way.